grid/src/Cabana_Grid_BovWriter.hpp

```diff
@@ -276,9 +278,11 @@ void writeTimeStep( ExecutionSpace, const std::string& prefix,
     }
     reorder_space_size.back() = owned_extents.back();
     IndexSpace<num_space_dim + 1> reorder_space( reorder_space_size );
-    auto owned_view = createView<value_type, Kokkos::LayoutRight, memory_space>(
+
+    auto owned_view = createView<value_type, Kokkos::LayoutRight, Kokkos::HostSpace>(
         array.label(), reorder_space );
-    reorderView( owned_view, owned_subview, reorder_space, ExecutionSpace() );
+
+    reorderView( owned_view, owned_subview, reorder_space, Kokkos::DefaultHostExecutionSpace{} );
```

Comment:
I don't see anything that would stop us from reordering on device, then mirroring after that kernel - what am I missing?

Reply:
@abisner I have segment fault error when encounter `node_halo->gather` and `Grid_Halo` CUDA test fails as well. Do you have same issue on `summit`

Reply:
@kwitaechong that may be unrelated - this should only affect the grid output tests